e2e/brickmux_test.go

```diff
@@ -255,6 +255,8 @@ func TestBrickMux(t *testing.T) {
 	r.Nil(err)
 	r.True(gd.IsRunning())
 
+	time.Sleep(5 * time.Second)
```

Comment:
What's the justification of waiting for 5 seconds here?

Reply:
@atinmu  Oh yeah, forgot to put a comment here. I have put a 5 sec sleep here just to give the bricks sometime to SignIn after a new gd2 instance has spawned.

Reply:
is there a way to remove this timeout, can we do something in during volume create?
IMO doing sleep in a test case is not a good idea.

Reply:
@Madhu-1  I know that doing a sleep is not the best idea here but the issue here is that the SignIn done by bricks sometime is not the quickest, sometimes they signin instantly, sometimes it takes time to signin and that's why we keep on seeing spurious failures in brickmux tests. Now the only solution to this is to make SignIn process as quick as possible.

@atinmu  I had a discussion with @aravindavk sometime back about whether we can somehow trigger a SignIn from GD2 side as soon as a glusterfsd spawns.